<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="RandomPushButton">
      <property name="text">
       <string>Random</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>774</width>
         <height>313</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="TeacherCntLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="CourseCntLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="RunDinicButton">
      <property name="text">
       <string>RunDinic</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="4">
     <widget class="QTableWidget" name="TeacherTableWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>老师姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>能胜任的课程</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="2" rowspan="4">
     <widget class="QTableWidget" name="CourseTableWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>课程名</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
